python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -136,18 +136,47 @@ def test_basic(ray_start_regular):
     compiled_dag.teardown()
 
 
-def test_multiple_returns_not_supported(ray_start_regular):
+def test_two_returns_first(ray_start_regular):
+    a = Actor.remote(0)
+    with InputNode() as i:
+        o1, o2 = a.return_two.bind(i)
+        dag = MultiOutputNode([o1])
+
+    compiled_dag = dag.experimental_compile()
+    res = ray.get(compiled_dag.execute(1))
+    assert res == [1]
+
+    compiled_dag.teardown()
+
+
+def test_two_returns_second(ray_start_regular):
+    a = Actor.remote(0)
+    with InputNode() as i:
+        o1, o2 = a.return_two.bind(i)
+        dag = MultiOutputNode([o2])
+
+    compiled_dag = dag.experimental_compile()
+    res = ray.get(compiled_dag.execute(1))
+    assert res == [2]
+
+    compiled_dag.teardown()
+
+
+def test_two_returns_three_actors(ray_start_regular):
     a = Actor.remote(0)
     b = Actor.remote(0)
+    c = Actor.remote(0)
     with InputNode() as i:
-        dag = a.return_two.bind(i)
-        dag = b.echo.bind(dag)
+        o1, o2 = a.return_two.bind(i)
+        o3 = b.echo.bind(o1)
+        o4 = c.echo.bind(o2)
+        dag = MultiOutputNode([o3, o4])
```

Comment:
Could you also add a test where o1 and o2 are read by the same actor task? Like:
```python
b.echo.bind(o1, o2)
```

Reply:
Done.